pom.xml

```diff
@@ -52,6 +53,17 @@
         <type>pom</type>
         <scope>import</scope>
       </dependency>
+      <dependency>
+        <groupId>org.eclipse.jetty.ee8</groupId>
+        <artifactId>jetty-ee8-bom</artifactId>
+        <version>${jetty.version}</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+      <dependency>
+        <groupId>org.hamcrest</groupId>
+        <artifactId>hamcrest</artifactId>
+      </dependency>
```

Comment:
As stated previously in https://github.com/jenkinsci/winstone/pull/341#discussion_r1593094412, this looks like a mis-merge of #375.

```suggestion
```